{{cookiecutter.project_slug}}/config/settings/local.py

```diff
@@ -69,6 +69,10 @@
 
     hostname, _, ips = socket.gethostbyname_ex(socket.gethostname())
     INTERNAL_IPS += [".".join(ip.split(".")[:-1] + ["1"]) for ip in ips]
+    {%- if cookiecutter.js_task_runner == 'Gulp' %}
+    _, _, ips = socket.gethostbyname_ex("node")
+    INTERNAL_IPS.extend(ips)
```

Comment:
So... This doesn't work, because you need the node container to be started, but the node container depends on th Django one:

https://github.com/pydanny/cookiecutter-django/blob/b48067c03494993933bd2eb0aa2e5afdf367f12f/%7B%7Bcookiecutter.project_slug%7D%7D/local.yml#L114-L115

I thought this dependency was needed for browsersync to work, but actually we might not need it, and we might be able to flip it (have django depends on node).

Anyway, we should probably catch [the error](https://github.com/pydanny/cookiecutter-django/pull/2865/checks?check_run_id=1758500147#step:4:1493) to not fail entirely in case `node` has an issue or is explicitely not started (like in tests):

```suggestion
    try:
        _, _, ips = socket.gethostbyname_ex("node")
        INTERNAL_IPS.extend(ips)
    except socket.gaierror:
        # The node container isn't started (yet?)
        pass
```